python/trim_osc/Portfile

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           active_variants 1.1
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        Zverik regional 7d473e463e1570bdd3b846c3d2539092c63bd98a
+version             20210417
+revision            0
+github.tarball_from archive
+
+name                trim_osc
+
+platforms           darwin
+maintainers         {@frankdean fdsd.co.uk:frank.dean} openmaintainer
+
+license             WTFPL
+
+description         Scripts for regional OSM extracts support
+
+long_description    It is hard to maintain OSM tile service with a small \
+                    server: you can't have a properly updated regional \
+                    extract or even run osm2pgsql on low memory. This \
+                    package constains some scripts that help.
+
+depends_lib-append  port:py${python.version}-lxml \
+                    port:py${python.version}-psycopg2 \
+                    port:py${python.version}-shapely
+
+checksums           rmd160  6e387ad3d613b3848778dd92df60d3c6296ac1b3 \
+                    sha256  c0eee61c14ef462d85e779bb108dedcb2054cf5ee453b2ec6a61039c0eabc648 \
+                    size    7041
+
+python.default_version  39
```

Comment:
I am pretty sure you need to have `python.default_version` before you ad dependencies that use `${python.version}. It does seem to work now, but that is just because the python PG sets a default (currently `39` but you don't want to rely on that) and it is using that.